mlflow/langchain/runnables.py

```diff
@@ -91,9 +94,21 @@ def _load_model_from_path(path: str, model_config=None):
         return _load_base_lcs(path, model_config)
     if model_load_fn == _CONFIG_LOAD_KEY:
         return _load_model_from_config(path, model_config)
+    if model_load_fn == _PICKLE_LOAD_KEY:
+        return _load_from_pickle(os.path.join(path, model_config.get(_MODEL_DATA_KEY)))
     raise MlflowException(f"Unsupported model load key {model_load_fn}")
 
 
+def _validate_path(file_path: Union[str, Path]):
+    # Convert file to Path object.
+    load_path = Path(file_path)
+    if not load_path.exists() or not load_path.is_dir():
+        raise MlflowException(
+            f"File {load_path} must exist and must be a directory in order to load model."
+        )
```

Comment:
/ai This error message sounds a bit confusing because it says file and directory. Plz improve it.